index.html

```diff
@@ -24,13 +25,153 @@
         <div class="social-icons">
             <ul class="social-icons">
                 <li><a href=""><img src="./images/twitter.svg" alt=""></a></li>
-                <li><a href=""><img src="./images/github.svg" alt=""></a></li>
+                <li><a href=""><img src="./images/github.png" alt=""></a></li>
                 <li><a href=""><img src="./images/Linkedin icon.svg" alt=""></a></li>
                 <li><a href=""><img src="./images/mail.svg" alt=""></a></li>
                 <li><a href=""><img src="./images/Vector.svg" alt=""></a></li>
             </ul>
         </div>
     </section>
+    <div class="bottom-sections">
+    <section class="projects-section">
+        <div class="card-works">
```

Comment:
- [OPTIONAL] One of the main requirement of this milestone is to make our design responsive up to `768px`. your hero and about sections are responsive up to `768px` but these project cards are not responsive. I will highly recommend you to make these cards responsive up to `768px`.